python/ray/serve/examples/doc/gradio-integration-parallel.py

```diff
@@ -0,0 +1,47 @@
+# __doc_import_begin__
+from ray import serve
+import ray
+import gradio as gr
+from ray.serve.gradio import GradioIngress
+from transformers import pipeline
+
+# __doc_import_end__
+
+# __doc_models_begin__
+@serve.deployment
+class Text_Generation_Model:
```

Comment:
```suggestion
class TextGenerationModel:
```

Python class names should use [PascalCase/CapWords style](https://peps.python.org/pep-0008/#class-names).